### ISSUE TITLE
GLIBCXX_3.4.22 requirement

### ISSUE THREAD

**Issue by YaLTeR** [YaLTeR/OpenAG#57]
Prevents working on e.g. Ubuntu 16.04
```
00000000      DO *UND*  00000000  GLIBCXX_3.4.22 _ZTINSt6thread6_StateE
00000000      DF *UND*  00000000  GLIBCXX_3.4.22 _ZNSt6thread6_StateD2Ev
00000000      DF *UND*  00000000  GLIBCXX_3.4.22 _ZNSt6thread15_M_start_threadESt10unique_ptrINS_6_StateESt14default_deleteIS1_EEPFvvE
```
Probably somewhere in the Discord integration.

**Comment by YaLTeR** [comment]
Upon further investigation, this appears whenever `std::thread` is used. This includes Discord as well as the update checker. https://github.com/YaLTeR/OpenAG/commit/205f5e0974c5ef5affe795639661b4a02ae544e9 added an option to link libstdc++ statically, so either that works out (at least one person was having issues trying to load `client.so` built like that), or I'll have to figure out a way of not using `std::thread`.

**Comment by JoelTroch** [comment]
Ubuntu 18.04 LTS might have a compatible updated version, need confirmation tho.

**Comment by YaLTeR** [comment]
The issue will be solved by compiling releases on Ubuntu 16.04.
